src/directive.js

```diff
@@ -26,9 +26,30 @@ export default {
       core.inputHandler(e, el)
     }
 
+    const compositionHandler = (e) => {
+      if (e.target !== el) {
```

Comment:
Is there any chance of the target not been the element?  This is a good fail safe, just curious since we are not doing this check for inputHandler

Reply:
Same check is in the `handler` function used for the `input` events. Looks like that was added for the case where the input element with the facade directive shares a parent with another input element (added in #41).